cardigan/stories/components/SeasonsHeader/SeasonsHeader.stories.tsx

```diff
@@ -14,29 +14,33 @@ const image: ImageType = {
   },
 };
 
-const headerProps = {
-  labels: { labels: [{ text: 'Article' }] },
+const season: Season = {
+  id: 'X84FvhIAACUAqiqp',
+  labels: [{ text: 'Article' }],
   title: 'What does it mean to be human, now?',
   start: new Date('2021-01-05T00:00:00.000Z'),
   end: new Date('2021-01-26T00:00:00.000Z'),
-  FeaturedMedia: <PrismicImage image={image} quality="low" />,
   standfirst: [
     {
       type: 'paragraph',
       text: 'Throughout history, pandemics have been powerful engines of change, exposing structural inequalities in the distribution of health and wealth.',
       spans: [],
     },
   ],
+  image: {
+    width: image.width,
+    height: image.height,
+    contentUrl: image.contentUrl,
+    alt: image.alt,
+    richCrops: {
+      '32:15': image,
+    },
+  },
+  body: [],
+  type: 'seasons',
 };
 
 const Template = args => <SeasonsHeader {...args} />;
 export const basic = Template.bind({});
-basic.args = {
-  labels: headerProps.labels,
-  title: headerProps.title,
-  start: headerProps.start,
-  end: headerProps.end,
-  standfirst: headerProps.standfirst,
-  FeaturedMedia: headerProps.FeaturedMedia,
-};
+basic.args = { season };
```

Comment:
I noticed Chromatic flagged image changes like we've had, if you'd like you could add:
```
basic.parameters = {
  chromatic: { diffThreshold: 0.2 },
};
```
to ignore those moving forward.